django_app/redbox_app/redbox_core/views.py

```diff
@@ -222,6 +224,10 @@ def post_message(request: HttpRequest) -> HttpResponse:
     files: list[File] = File.objects.filter(core_file_uuid__in=doc_uuids, user=request.user)
     llm_message.source_files.set(files)
 
+    for file in files:
```

Comment:
I _think_, according to core, this is just all files for that user, we don't need to use a response from core for that

Reply:
Is it not the ones returned as relevant by the LLM chain `core_api/src/routes/chat.py#L150-L151`?

Reply:
I believe you're right, Rachael - it's just the "sources", i.e. the relevant files.